styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -143,6 +147,8 @@ void tick() {
                 ConcurrentHashMap::new,
                 counting()));
 
+    currentResourceUsage.forEach(stats::resourceUsage);
```

Comment:
A bit disorienting to read statements in a program without any verbs. Not to address in this PR, just complaining.